babybear/src/main/java/frc/robot/subsystems/WristSubsystem.java

```diff
@@ -0,0 +1,172 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: WristSubsystem.java
+// Intent: Subsystem to model the wrist.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.subsystems;
+
+import frc.robot.common.*;
+import frc.robot.Constants;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.SparkMaxRelativeEncoder;
+import com.revrobotics.CANSparkMax.*;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+/**
+ * A class intended to model the intake - 1 Geared neo550 to move the arm to a up or down position.
+ */
+public class WristSubsystem extends SubsystemBase {
+   /**********************************************************************
+    CONSTANTS
+    ************************************************************************/
+    // Wrist gear reduction
+    private static final double wristGearReduction = 48.0/1.0;
+    
+    private static final double TICKS_PER_DEGREE = 360/42 * wristGearReduction; //TODO SET THIS encoder ticks per arm degree (will corelate with gearbox)
+    private static final double TOLERANCE = 5; // Encoder Tolerence, raise or lower if it bounces or doesn't reach the target.
+
+    // TODO - use something less than 1.0 for testing
+    private static final double wristMotorSpeedReductionFactor = 1.0;
+
+    
+
+    /**********************************************************************
+    MEMBERS
+    ************************************************************************/
+    private CANSparkMax wristMotor = new CANSparkMax(Constants.wristMotorCanID, MotorType.kBrushless);
+    private SparkMaxPIDController wristPidController;
+    private RelativeEncoder wristEncoder;
+    private double kP, kI, kD, kIz, kFF, kMaxOutput, kMinOutput, maxRPM, maxVel, minVel, maxAcc, allowedErr;
+    private boolean motorInitializedForSmartMotion = false;
+
+    private boolean isWristMotorInverted = false;
+    private double requestedWristMotorSpeed = 0.0;
```

Comment:
So I think there are 3 motors planned for the 'wrist subsystem' if I'm not mistaken.
1 - neo motor (spark max controller) for the wrist movements - this will have encoder and will have pickup and stow positions (plus maybe some others)
2 - bag motors (STX controller) that are geared the same that will drive the intake/outtake 'cube picker' 

so seems like there is more needed here especially around what needs to happen for the 'cube picker' dual bag motors